gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -64,10 +64,10 @@ def get_shared_images
   end
 
   def get_public_images
+    filters = @options.public_images_filters
```

Comment:
In case you are familiar with rspec, it would be great to have a test for it.
There is no `refresh_parser_spec.rb` yet, so you'd have to create it.
Then only add a test for `get_public_images` which stubs `get_images` and expects `@options.public_images_filters` to return an array that matches replaced one.